scripts/tests-cli.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export PYTHONPATH=`pwd`/oblib
+export PYTHONPATH=`pwd`
```

Comment:
Can we see if the test succeeds without this export statement?

Reply:
Dang, hoped that would test OK.  I asked a friend about the import issues and he said that export PYTHONPATH is risky, it could remove references to other python packages, but the behavior may be platform dependent.  It would be best is we didn't touch the python path.

Reply:
I guess put it back in to get this PR finished, and let's open a new issue to address this line of code.